app/components/delete-node-modal/component.ts

```diff
@@ -0,0 +1,15 @@
+import Component from '@ember/component';
+import defaultTo from 'ember-osf-web/utils/default-to';
+import randomScientist from 'ember-osf-web/utils/random-scientist';
+
+export default class DeleteNodeModal extends Component.extend({
+    didReceiveAttrs(this: DeleteNodeModal, ...args) {
+        this._super(...args);
+        this.set('scientistInput', '');
+        this.set('scientistName', randomScientist());
+    },
```

Comment:
This can go in the class body. See: https://github.com/CenterForOpenScience/ember-osf-web/blob/0fcbb763fbef29b5cf8525dc1e297bdd3069c921/app/components/file-browser/component.ts#L197-L202